### PR TITLE
Document command syntax, flags, and options

### PR DESCRIPTION
**Pull request by gaffneyd4**
Resolves (at least partially): https://github.com/kjvalencik/ghshare/issues/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,80 @@
...
 
-```sh
-ghshare encrypt -h https://example.com/api/v3 ...
+Basic form of encryption command:  
+
+```
+ghshare encrypt --input <plaintextFileName> --output <ciphertextFileName> --recipient <receiversGithubUsername>
+```
+
+> :information_source: Each of the receiver's GitHub public keys (available here: https://api.github.com/users/receiversGithubUsername/keys) will be used to encrypt the input file.
```
Comment (kjvalencik) [inline README.md:30]: Technically, they only encrypt the encryption key and not the file, but I don't think that's a meaningful difference for users.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,80 @@
...
+`--small`, `-s`              | Encrypt without header for small data, OpenSSL compatible
+`--version`, `-V`            | Prints version information
+
+#### options
+
+ flag               | param        | default                | required | purpose
+--------------------|--------------|------------------------|----------|---------
+`--host`, `-h`      | `<host>`     | https://api.github.com | `false`  | Endpoint to query for recipient's public keys
+`--input`, `-i`     | `<path>`     |                        | `true`   | Path to file to be encrypted
+`--output`, `-o`    | `<path>`     |                        | `true`   | File path to write encrypted data to
```
Comment (kjvalencik) [inline README.md:47]: Neither of these are required. When not provided, they will use `stdin` and `stdout` respectively.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,80 @@
...
+`--small`, `-s`              | Decrypt small data without a header, OpenSSL alternative
+`--version`, `-V`            | Prints version information
+
+#### options
+
+ flag            | param    | required | purpose
+-----------------|----------|----------|---------
+`--input`, `-i`  | `<path>` | `true`   | Path to file to be decrypted
+`--key`, `-k`    | `<path>` | `true`   | Path to private key file used for decryption
+`--output`, `-o` | `<path>` | `true`   | File path to write decrypted data to
```
Comment (kjvalencik) [inline README.md:75]: Same as above.

**Comment by kjvalencik** [comment]
Thanks so much for this!!

**Review by kjvalencik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gaffneyd4) [reply README.md]: [Made the statement more generic](https://github.com/kjvalencik/ghshare/pull/13/commits/73c4aef3b3b47dc07501abdb4fa8c4c0b8006470#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31)

File: README.md (hunk 2 above)
Comment (gaffneyd4) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (gaffneyd4) [reply README.md]: Fixed

**Review by kjvalencik — APPROVED** [review]

**Pull request merged**